lib/nerves_runtime/application.ex

```diff
@@ -1,22 +1,26 @@
 defmodule Nerves.Runtime.Application do
-  # See http://elixir-lang.org/docs/stable/elixir/Application.html
-  # for more information on OTP Applications
```

Comment:
I agree with the deletion of the boiler plate comments, but could you separate those and the other cleanup changes to separate PRs for next time?